### PR TITLE
 Update solidity-support: block.coinbase and block.difficulty

### PR DESCRIPTION
**Pull request by DZGoldman**

### CODE REVIEW

**Review by sammyfilly — COMMENTED** [review]
allow changes

File: arbitrum-docs/solidity-support.mdx [hunk 1]
```diff
@@ -7,7 +7,7 @@ Arbitrum Nitro chains are Ethereum compatible, and therefore allow you to trustl
 Although Arbitrum supports Solidity code, there are differences in the effects of a few operations, including language features that don't make much sense in the Layer 2 context:
 
 - `blockhash(x)` returns a cryptographically insecure, pseudo-random hash for `x` within the range `block.number - 256 <= x < block.number`. If `x` is outside of this range, `blockhash(x)` will return `0`. This includes `blockhash(block.number)`, which always returns `0` just like on Ethereum. The hashes returned do not come from L1.
-- `block.coinbase` returns zero
+- `block.coinbase` returns the address of the L2 transaction's L1 batch poster
```
Comment (mds1) [inline arbitrum-docs/solidity-support.mdx:10]: Will the batch poster always be `0xA4b000000000000000000073657175656e636572`? I noticed this error in the docs while working on [evm-diff](https://github.com/mds1/evm-diff) and documented this as a question [here](https://github.com/mds1/evm-diff/issues/17). I came here to open an issue about it, but than found this PR.

There's a few other questions in there of things I couldn't find in the docs, or places where the docs seem inaccurate.

Comment (mds1) [reply arbitrum-docs/solidity-support.mdx]: Thanks!

Does `tx.origin` also need to be added? I'm not sure if `tx.origin` address is also aliased for L1-to-L2 "retryable ticket" transactions, or if only `msg.sender` is aliased

**Review by chrisco512 — APPROVED** [review]

**Review by TucksonDev — APPROVED** [review]

**Review by tsahee — APPROVED** [review]
LGTM

**Pull request merged**